src/main/java/picard/fingerprint/CrosscheckFingerprints.java

```diff
@@ -289,6 +289,11 @@
             "Should only be used with SECOND_INPUT. ", optional = true)
     public File SECOND_INPUT_SAMPLE_MAP;
 
+    @Argument(doc = "A tsv with two columns representing the individual with which each sample is associated.  When this sample is specified, expectations for matches will be based " +
+            "on the equality or inequality of the individual associated with two samples, as opposed to the sample ids themselves.  Samples which are not included in this file " +
+            "will have their sample id used as their individual id", optional = true)
```

Comment:
- what happens if this points one sample onto another sample that isn't mapped? 
- is the map the same for samples in INPUT and SECOND_INPUT?

Reply:
Be more specific about the format of the file, since it currently isn't specified which column is which and there are no column headers mentioned.

Reply:
Not sure I understand the first question.  This maps samples to individuals, not samples to other samples.  So if only one sample maps to a particular individual, that is not a problem, it just means that we will never expect this sample to match when checked against any of the other samples.

Yes, the same map is used for both INPUT and SECOND_INPUT.   Since this map represents the relationship between samples and actual individuals, there is no need to distinguish between samples in INPUT and SECOND_INPUT when enumerating these.  If the same sample id occurs in both INPUT and SECOND_INPUT, it would be associated to the same individual.  Note that we can actually use INPUT_SAMPLE_MAP/SECOND_INPUT_SAMPLE_MAP and SAMPLE_INDIVIDUAL_MAP together, so that we can actually map sample --> sample alias --> individual if we want/need to. 

And added more specifics on the file format and details on the usage.

Reply:
I guess I meant what if the map points a sample at an individual that is also another sample? these are all strings...so you can have
```
NA12878 -> GM12878 (in the map)
GM12878 (a sample, not in the map) 
```
it should be made clear that in this case, NA12878 and GM12878 will be considered to be the same "individual".

Reply:
Also, should make sure that it is clear that "GROUP_BY=SAMPLE" means AFTER mapping.

Reply:
Actually....it doesn't seem that you did that. Should there also be a GROUP_BY=INDIVIDUAL that enables this?

Reply:
Ahh, I see.  Added some documentation to clarify that this is the case, and logging a warning when this happens

Reply:
Right, this has no effect on the group_by operation currently, just on the match/no match expectations.  A `CROSSCHECK_BY=INDIVIDUAL` option could be added, but I think it would be a little more involved, for example I think a new `individual` field would need to be added to the `FingerprintIdDetails` class, and some handling of setting that field in different scenarios.  I also think that use case is much rarer than the one this PR is trying to solve, it seems rather unusual to me to want to actually combine all the data from different samples which are from the same individual into a single fingerprint, and then check that against other "combined" fingerprints composed of multiple different samples from other individuals.  In fact, I kind of think that option would be more often invoked incorrectly by mistake than it would be used intentionally correctly.  So I'm not sure that it is worth adding here.